### ISSUE TITLE
`HelpText` should have `aria-expanded`

### ISSUE THREAD

**Issue by Barsnes** [digdir/designsystem#991]
### Description of the bug

Currently the help text does not have `aria-expanded`

### Steps To Reproduce

Use The `HelpText`

**Comment by mimarz** [comment]
This was reported via slack
